draft-ietf-rats-corim.md

```diff
@@ -1194,7 +1195,22 @@ applies to all measurements in the triple, including those in `measurement-value
 
 ## Extensibility {#sec-extensibility}
 
-[^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/10
+CoRIM is described using CDDL {{-cddl}} which includes extensibility support for "sockets" (a.k.a., map structures)
+and data types. CDDL socket extensions have the form `($$NAME-extension)` where "NAME" is the name of the map.
```

Comment:
$$foo is a an extension point for a group-choice
$bar is an extension point for a type choice

Not sure if that level of detail is relevant here, but that is less ambiguous to me.

Reply:
@henkbirkholz to work on this PR